compiler/rustc_middle/src/query/mod.rs

```diff
@@ -198,7 +198,7 @@ rustc_queries! {
     ///
     /// This can be conveniently accessed by methods on `tcx.hir()`.
     /// Avoid calling this query directly.
-    query hir_attrs(key: hir::OwnerId) -> &'tcx hir::AttributeMap<'tcx> {
+    query hir_attr_map(key: hir::OwnerId) -> &'tcx hir::AttributeMap<'tcx> {
```

Comment:
Just to double-check, this was renamed to make way for `Map::attrs` becoming `TyCtxt::hir_attrs`, right?